aspnetcore/migration/22-to-30.md

```diff
@@ -1048,18 +1048,13 @@ For more information, see [Breaking change in `AddAuthorization(o =>`) overload
 
 ## Identity UI
 
-Complete the following steps:
+Identity UI updates for ASP.NET Core 3.0
 
 * Add a package reference to [Microsoft.AspNetCore.Identity.UI](https://nuget.org/packages/Microsoft.AspNetCore.Identity.UI).
-* If the app isn't already using Razor Pages, add a call to `MapRazorPages`. See [Razor Pages](#razor-pages) in this document.
-* Prior to ASP.NET Core 3.0, the default UI framework was Bootstrap 3. The UI framework could be set to Bootstrap 4 with the following code:
+* Apps that don't use Razor Pages must call `MapRazorPages`. See [Razor Pages](#razor-pages) in this document.
+* The ASP.NET Core 2.x templates used Bootstrap 3. The ASP.NET Core 3.0 templates use Bootstrap 4. The following code shows an example of how to update to Bootstrap 4:
```

Comment:
> The ASP.NET Core 2.x templates used Bootstrap 3. The ASP.NET Core 3.0 templates use Bootstrap 4.

I think this puts too much emphasis on the project templates, rather than the change in `AddDefaultUI`. IIRC, 2.1 didn't have the option at all, so only Bootstrap 3 was available, but the 2.2 projects overrode to use Bootstrap 4 with `.AddDefaultUI(UIFramework.Bootstrap4)`.

> The following code shows an example of how to update to Bootstrap 4:

The code this refers to is only valid in 2.2, but now I think it reads as though it's an instruction for 3.0.

---

Perhaps we could just mention a change to the default and point to the linked announcement for the details?

Reply:
> Perhaps we could just mention a change to the default and point to the linked announcement for the details?

Yes, do that

Reply:
@Rick-Anderson How's that?

Reply:
Fantastic